.mrconfig_production

```diff
@@ -34,7 +34,9 @@ checkout =
 
 [mnc_python]
 checkout =
-    git clone https://github.com/ovro-lwa/mnc_python.git
+    git pull --tags origin main
+    git checkout 0.8.0
+    git checkout -- .
```

Comment:
what is this line doing? isn't 'git checkout 0.8.0 enough?

Reply:
blindly copying code w/out understanding. :/ :) I should have added a comment here so I'll go hunt down why this line is necessary.